includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -724,24 +724,39 @@ public function ajax_get_cart_details() {
 	public function ajax_get_shipping_options() {
 		check_ajax_referer( 'wc-stripe-payment-request-shipping', 'security' );
 
-		try {
-			// Set the shipping package.
-			$posted = filter_input_array(
-				INPUT_POST,
-				array(
-					'country'   => FILTER_SANITIZE_STRING,
-					'state'     => FILTER_SANITIZE_STRING,
-					'postcode'  => FILTER_SANITIZE_STRING,
-					'city'      => FILTER_SANITIZE_STRING,
-					'address'   => FILTER_SANITIZE_STRING,
-					'address_2' => FILTER_SANITIZE_STRING,
-				)
-			);
+		$shipping_address = filter_input_array(
+			INPUT_POST,
+			array(
+				'country'   => FILTER_SANITIZE_STRING,
+				'state'     => FILTER_SANITIZE_STRING,
+				'postcode'  => FILTER_SANITIZE_STRING,
+				'city'      => FILTER_SANITIZE_STRING,
+				'address'   => FILTER_SANITIZE_STRING,
+				'address_2' => FILTER_SANITIZE_STRING,
+			)
+		);
 
-			$this->calculate_shipping( apply_filters( 'wc_stripe_payment_request_shipping_posted_values', $posted ) );
+		$data = $this->get_shipping_options( $shipping_address );
+		wp_send_json( $data );
+	}
 
+	/**
+	 * Gets shipping options available for specified shipping address
+	 *
+	 * @param array $shipping_address Shipping address.
+	 *
+	 * @return array Shipping options data.
+	 * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag
+	 */
+	public function get_shipping_options( $shipping_address ) {
+		try {
 			// Set the shipping options.
-			$data     = array();
+			$data = array();
+
+			// Remember current shipping method before resettig.
```

Comment:
Sorry for nitpicking, should this be resetting?

Reply:
Thanks @luizreis for catching this! Sorry, I'm making too many typos in the code 😕Will try to find a spell checking solution for VS Code. I think I have something but apparently it doesn't work well.

Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1191/commits/0b668daa2a703f8607d62964248cc033bf638345